### ISSUE TITLE
Make static pages configurable for each deployment

### ISSUE THREAD

**Issue by AngelMatilla** [chemins-de-la-transition/semapps#610]
In the current repo there's a list of static pages, including the home page that are hardcoded as js and css files. 
You can find most of them (maybe all) here: https://github.com/chemins-de-la-transition/semapps/tree/master/frontoffice/src/pages

It would be necessary for the Regen4all project to be able to modify those pages (including the links to other pages). 

My proposal (does not need to be the final solution) is that we make the URLs to the static pages configurable and that we locate the js and css files outside of the main repo (in order not to have to fork for each new project / implementation. Maybe not all files need to be located outside of the repo, and only those with content are the ones that can be removed.

A more elegant but also complex solution would be to make the pages configurable within the backoffice, but it might be too much effort for the desired result.

**Comment by srosset81** [comment]
The AboutPage is the only page specific to CDLT.
All other pages (for example https://lescheminsdelatransition.org/Page/mentions-legales/show) are markdown pages which can be edited on the backoffice.

**Comment by AngelMatilla** [comment]
What about the texts on the following pages: 
* Login page: https://lescheminsdelatransition.org/login
* Signup page: https://lescheminsdelatransition.org/login?signup=true
* Mybookmarks
* myevents
* myorganizations
* etc
I imagine these will be localized with [this other ticket ](https://github.com/chemins-de-la-transition/semapps/issues/586)

In any case, it would be important to be able to customize the Home Page and About Page for our instance, so either we decide to fork the repo, or we create another sub-repo just with the pages

**Comment by VincentFarcy** [comment]
> What about the texts on the following pages:
> 
> * Login page: https://lescheminsdelatransition.org/login
> * Signup page: https://lescheminsdelatransition.org/login?signup=true
> * Mybookmarks
> * myevents
> * myorganizations
> * etc
>   I imagine these will be localized with [this other ticket ](https://github.com/chemins-de-la-transition/semapps/issues/586)

yes, all the my...pages only contain a unique title to be translated
And the login/signup page a unique text to be translated

**Comment by Tonfa** [comment]
That's what I was trying to say in the meeting because I'm currently
working on internationalization I didn't see static pages besides homepage
and about (and yeah I'm translating all the other ones)

**Comment by srosset81** [comment]
> In any case, it would be important to be able to customize the Home Page and About Page for our instance, so either we decide to fork the repo, or we create another sub-repo just with the pages

These two pages are full React pages, so it will require a developer to customize them.
The easiest solution for now would be to put the new HomePage and AboutPage in the `/deploy/frontoffice/app/src/pages` directory. When the Docker container is built, it will [automatically overwrite](https://github.com/chemins-de-la-transition/semapps/blob/master/deploy/frontoffice/Dockerfile#L29) the default CDLT ones.
